src/pages/test-report/cat-cpc/components/cpc-end-test-modal/components/modal-result-item/modal-result-item.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'modal-result-item',
+  templateUrl: 'modal-result-item.html',
+})
+export class ModalResultItemComponent {
+
+  @Input()
+  label: string;
+
+  @Input()
+  score: number;
+
+  @Input()
+  isPass: number;
+
+  getOutcomeIcon(): string {
+    const passImage = '../../../../../../../assets/imgs/greenCorrectAnswer.png';
+    const failImage = '../../../../../../../assets/imgs/redWrongAnswer.png';
```

Comment:
in other places where an icon is displayed the HTML references just 'assets/imgs/greenCorrectAnswer.png' for example, does this still work if the path is:

 const passImage = 'assets/imgs/greenCorrectAnswer.png';

Reply:
changed